src/community/spatialjson/pom.xml

```diff
@@ -28,6 +28,11 @@
       <groupId>org.geoserver.web</groupId>
       <artifactId>gs-web-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>30.1-jre</version>
```

Comment:
```suggestion
```
Several other modules use guava and the version is managed in the parent POM.
Adding a version here is bound to give problems when it is upgraded in the parent and forgotten here

Reply:
Yes, I though about that but wasn't quite sure how do deal with it correctly. Thanks for clarifying.